src/lib/fpti.js

```diff
@@ -18,7 +18,7 @@ export const sendBeacon = (src : string, data : FptiVariables | LegacyVariables)
 
   query = query.join('&');
 
-  const beaconImage = new window.Image();
+  const beaconImage = document.createElement('img');
```

Comment:
in case anyone is curious about this change, there's almost no difference between the 2: https://stackoverflow.com/questions/6241716/is-there-a-difference-between-new-image-and-document-createelementimg

Reply:
Thanks @songz. I changed this so that we could use the existing image mock functions in the test suite.